### PR TITLE
Thumbnails improvements

### PR DESCRIPTION
**Pull request by wb9688**
From https://github.com/lxde/libfm/pull/25

### CODE REVIEW

**Comment by wb9688** [comment]
I think the thumbnails are a bit too big and the default threshold a bit too high, but that should be easy to fix. I want to test it a bit more though.

File: src/gtk/fm-thumbnail.c [hunk 1]
```diff
@@ -193,6 +193,26 @@ static GObject* scale_image(GObject* ori_pix, int new_width, int new_height)
...
+static int get_colorspace(GObject* image)
+{
+    return (int)gdk_pixbuf_get_colorspace(GDK_PIXBUF(image));
+}
+
+static int get_bits_per_sample(GObject* image)
+{
+    return gdk_pixbuf_get_bits_per_sample(GDK_PIXBUF(image));
+}
+
```
Comment (wb9688) [inline src/gtk/fm-thumbnail.c:215]: These functions aren't really necessary

**Comment by wb9688** [comment]
We also have to add the settings added here in Cephalopod's preferences

**Comment by damianatorrpm** [comment]
Yeah I will add them, I would add the to gsettings once this PR is merged

File: src/gtk/fm-thumbnail.c (hunk 1 above)
Comment (wb9688) [reply src/gtk/fm-thumbnail.c:215]: Actually, they are, because libfm doesn't have a dependency on GDK Pixbuf.

**Pull request merged**